README.md

```diff
@@ -1 +1,38 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+# Portfolio-setup-and-mobile-version-skeleton
+
+## Description the project
+This project is to setup a portfolio of mobile version skeleton main page
+
+## Built With
+* HTML and CSS
+
+## Authors
+
+### 👤 Author1
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterbhandle](https://twitter.com/twitterhandle)
+- Linkedln: [Linkedln](https://linkedin.com/in/linkedinhandle)
+
+### 👤 Author2
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterbhandle](https://twitter.com/twitterhandle)
+- Linkedln: [Linkedln](https://linkedin.com/in/linkedinhandle)
+
+## 🤝 Contributing
```

Comment:
- [ ] Good job adding README file to your project, Kindly add your contact details here to make your project more professional

Reply:
I have updated my contact details